src/rad/resources/schemas/source_catalog-1.0.0.yaml

```
@@ -0,0 +1,23 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/source_catalog-1.0.0
+
+title: Source catalog generated by the Source Catalog Step.
+
+datamodel_name: SourceCatalogModel
+
+archive_meta: None
+type: object
+properties:
+  source_catalog:
+    title: Source Catalog
+    description: |
+       Photometry and astrometry computed in the Source Catalog Step.
+    tag: tag:astropy.org:astropy/table/table-1.*
+
+
+required: [meta, source_catalog]
+flowStyle: block
+propertyOrder: [meta, source_catalog]
```

```suggestion
required: [source_catalog]
flowStyle: block
propertyOrder: [source_catalog]
```
This will fix the broken tests for now.

---

I wouldn't have figured this out quickly without you. Thank you 🙏🏻